src/styles/_mixins.scss

```diff
@@ -1,20 +1,3 @@
-@mixin xs {
-    @media (max-width: 600px) {
-      @content;
-    }
-  }
-  @mixin sm {
-    @media (max-width: 960px) {
-      @content;
-    }
-  }
-  @mixin md {
-    @media (min-width: 960px) {
-      @content;
-    }
-  }
-  @mixin lg {
-    @media (min-width: 1264px) {
-      @content;
-    }
+  @mixin fade-transition {
+    transition: all 0.3s ease-in-out;
```

Comment:
be careful with transition for `all` elements - it may lead to unexpected behavior e.g. while RWD related styles are applied (resizing window) etc. 😉